content/ru/docs/intro/quickstart.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Краткое Руководство"
+description: "How to install and get started with Helm including instructions for distros, FAQs, and plugins."
+weight: 1
+aliases: ["/docs/quickstart/"]
```

Comment:
You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.